src/hadoop-jobhistory/deploy/node-label.sh.template

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
@YitongFeng  To ensure, whether this file could be removed or not.

Reply:
could removed